src/buttons/src/IconButton.js

```diff
@@ -78,7 +78,7 @@ IconButton.propTypes = {
   /**
    * The intent of the button.
    */
-  intent: PropTypes.oneOf(['none', 'success', 'warning', 'danger']),
+  intent: PropTypes.oneOf(['none', 'success', 'warning', 'danger', 'info']),
```

Comment:
I don't think many of these actually support styles for an `info` intent, as it's considered the same as `none`

Reply:
+1. The style for `info` is the same as `none`. Instead of adding `info` back, should we clean up where we have `info`?